homework_1_collections/Nucleo.py

```diff
@@ -0,0 +1,135 @@
+# Transcribes DNA to RNA.
+# dna_seq - string, DNA sequence.
+def transcribe(dna_seq):
+    x = list(dna_seq)
+    for i in range(len(x)):
+        if x[i] == 'T':
+            x[i] = 'U'
+        if x[i] == 't':
+            x[i] = 'u'
+    x = (''.join(x))

```

Comment:
Можно проще
```suggestion
    x = x.replace('T', 'U').replace('U', 'T')
```
Хотя твой вариант будет всё равно работать чуток быстрее